mappings/net/minecraft/world/RangeProvider.mapping

```diff
@@ -0,0 +1,23 @@
+CLASS net/minecraft/class_6497 net/minecraft/world/RangeProvider
```

Comment:
It does appear to be used in `world`, but I believe `util` is a better place given its obf name is next to `util.Lazy`, like this is `ahf` and `Lazy` is `ahg`.

In addition, this is not a `Provider` but just a range itself. It should be `Range` `Bounds` or `InclusiveRange`

Reply:
I chose that name and package because the syntax seemed like the one in worldgen int provider - but yeah, this is better.

Reply:
Decided to put in util/dynamic

Reply:
Isn't util/dynamic for Codec things? I mean, this has a codec, but other than that, it doesn't do anything with that.

Reply:
I think we currently have `GlobalPos` there. Can refactor things if they are confusing.